Moderation-Policy.md

```diff
@@ -1,5 +1,5 @@
 # Moderation Policy
 
 The Node.js Moderation policy has moved to
-<https://github.com/nodejs/admin/blob/master/Moderation-Policy.md> Please update
+<https://github.com/nodejs/admin/blob/main/Moderation-Policy.md> Please update
```

Comment:
nit: In other instances, we've changed `master` with `HEAD`, so it's not dependent on the name of the default branch. (it seems quite unlikely the name of the default branch changes again, so feel free to ignore)
```suggestion
<https://github.com/nodejs/admin/blob/HEAD/Moderation-Policy.md> Please update
```